sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -613,12 +670,45 @@ def import_key(self, name, key, **kwargs):
         )
         return KeyVaultKey._from_key_bundle(bundle)
 
+    @distributed_trace
+    def release_key(self, name, target, version=None, **kwargs):
```

Comment:
Maybe it's different for Python, but in past archboard meetings they wanted `name` and `version` to be kept in the same order. I realize Python doesn't support overloading, though, so maybe this is acceptable. I would talk with your architects if you're not sure.

Reply:
That's good to know; I'll open an issue to discuss this before the next release. We can't have a required parameter after an optional one, which means we'd need to lead with `target` if we keep `name` and `version` adjacent -- but that feels like an odd ordering since the key is being acted on. @chlowell, do you think this is a good enough signature for the beta?

Reply:
I'm okay with this signature. We'll get more opinions and hear what the architects think before the stable release.